Widgets/RoomView.cpp

```diff
@@ -118,8 +126,9 @@ void RoomView::paintInstances(QPainter& painter, Room* room) {
     int yoff = 0;
 
     const ProtoModel* spr = GetObjectSprite(inst.object_type());
-    if (spr == nullptr)
+    if (spr == nullptr) {
       imgFile = "object";
+    }
     else {

```

Comment:
Either remove those braces you added or put the else on the same line. Either will be logically equivalent so it doesn't matter, but please do something other than what you have.